app/util/JavaUtils.scala

```diff
@@ -0,0 +1,15 @@
+package util
+
+object JavaUtils {
+
+  def autoClose[A <: AutoCloseable](closeable: A)(fun: (A) ⇒ Unit)(exHandler: Exception => Unit): Unit = {
```

Comment:
Is this really needed if we're only using it in one place?

Reply:
I think that's fine. It's a very abstract method we could need if we extend some features.